swri_roscpp/include/swri_roscpp/node_handle.h

```diff
@@ -890,6 +918,14 @@ bool getParam(swri::NodeHandle& nh,
   return res;
 }
 
+template<typename T>
+void setParam(swri::NodeHandle& nh,
+  const std::string name,
```

Comment:
Should we pass ```name``` by reference?

Reply:
I think so, the NodeHandle setParam function passes name by reference.

Reply:
Can you update it to pass by reference then?

Reply:
It looks like it has been updated. This comment is on an old version, the current one seems to have it